.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Issue/Task Number:
+
+# Overview
+
+Describe what your Pull Request is about in a few sentences. This is a global description that should tell people what your PR is all about. Don't go too much into details here.
+
+# Changes
+
+- Change 1
+- Change 2
+- ...
+
+# Implementation Details
```

Comment:
What do you think about merging the changes/implementation sections? They seem to be communicating the same pieces of information (how this thing was implemented, decisions made, trade-offs, etc..).

It looks like all of the recent PRs in the e-wallet repo are pretty much merging these sections as well.